frontend/website/src/routes/Header.svelte

```diff
@@ -34,9 +46,58 @@
 		if (isSignedIn) {
 			signedIn.set(false);
 		} else {
-			goto('/signin');
+			signin = true;
 		}
 	}
+
+	function handleLogin(event: Event): void {
+		event.preventDefault();
+		signedIn.set(true);
+	}
+
+	function validatePassword(password: string) {
+		const length = password.length >= 12;
+		const upperLower = /[a-z]/.test(password) && /[A-Z]/.test(password);
+		const lettersNumbers = /[a-zA-Z]/.test(password) && /\d/.test(password);
+		const specialChar = /[!@#?\]]/.test(password) && !/[<>]/.test(password);
+
+		return {
+			length,
+			upperLower,
+			lettersNumbers,
+			specialChar,
+			all: length && upperLower && lettersNumbers && specialChar
+		};
+	}
+
+	function validateEmail(email: string) {
+		return email.includes('@');
+	}
```

Comment:
No, fix this that is not all an email requires at all.

`^[a-zA-Z\d_+&*-]+(?:\\.[a-zA-Z\d_+&*-]+)*@(?:[a-zA-Z\d-]+\.)+[a-zA-Z]{2,7}$` is an basic example of email validation.